app/presenters/organisations/show_presenter.rb

```diff
@@ -110,6 +108,16 @@ def contact_line(line)
       ""
     end
 
+    def make_link(link)
+      return link_to(make_link_text(link["description"]), link["link"]) if link["link"].length.positive?
+      nil
+    end
+
+    def make_link_text(text)
+      return I18n.t('organisations.foi.contact_form') unless text.length.positive?
```

Comment:
Can you do `text.empty?`. Not sure how the link descriptions work - are they enforced or could it be `nil`?

Reply:
Also I think this could be if you wanted to avoid having the additional line that returns `text`:

```
text.length.positive? ? text : I18n.t('organisations.foi.contact_form')
```